File: snippets/php/handling_the_webhook_response_in_your_application.php
<?php

// this is a stand-in for the code you'd use to write to your own database
function subscribeUser($id) {}
function unsubscribeUser($id) {}

if ($_SERVER['REQUEST_METHOD'] === 'POST') {
  $type = $_POST['type'];
  $id = $_POST['data']['id'];

  if ($type === 'subscribe') {
    subscribeUser($id);
  } else if ($type === 'unsubscribe') {
    unsubscribeUser($id);
  }
}

File: snippets/python/handling_the_webhook_response_in_your_application.py
from flask import Flask,request
from operator import itemgetter

app = Flask(__name__)

@app.route('/', methods=['POST'])
def index():
  reqType= itemgetter('type')(request.get_json())
  reqData= itemgetter('data')(request.get_json())
  if reqType == "subscribe" :
   fakeDB.subscribeUser(reqData)
  elif reqType == "unsubscribe" :
   fakeDB.unsubscribeUser(reqData['id'])

if __name__ == '__main__':
    app.run(debug=True, host='0.0.0.0')